src/main/java/nl/thehyve/whereabouts/controllers/InstanceController.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2019 The Hyve
+ * This file is distributed under the MIT License (see accompanying file LICENSE).
+ */
+
+package nl.thehyve.whereabouts.controllers;
+
+import nl.thehyve.whereabouts.domains.Instance;
+import nl.thehyve.whereabouts.exceptions.InstanceNotFoundException;
+import nl.thehyve.whereabouts.repositories.InstanceRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+@RestController
+public class InstanceController {
+
+    private final InstanceRepository instanceRepository;
```

Comment:
I would like to discuss the usage of services and representation/Data Transfer Objects (DTO).
The idea is that:
- controllers handle http requests and to mapping to/from json and http responses (status code and message). In controllers, the data is transferred using simple Java objects (sometimes called POJOs for Plain Old Java Object)
- services handle transactions on data, like fetching, saving, updating. Actions are transactional, data is still received and returned as POJOs.
- repositories handle database communication ((de)serialising data, executing queries and statements). We use Hibernate for this, which means Entity classes, which are objects wrapped with interceptors that extend actions with database related behaviour. 
- For mapping between DTOs and Entity objects, we can use [mapstruct](http://mapstruct.org/; no need to write a lot of straighforward mapping code).